Framework/PythonInterface/plugins/algorithms/StringToPng.py

```diff
@@ -40,10 +40,7 @@ def PyExec(self):
             import matplotlib
         except ImportError:
             ok2run = "Problem importing matplotlib"
-            from distutils.version import LooseVersion
 
-            if LooseVersion(matplotlib.__version__) < LooseVersion("1.2.0"):
-                ok2run = "Wrong version of matplotlib. Required >= 1.2.0"
         if ok2run != "":
             raise RuntimeError(ok2run)
```

Comment:
I think we don't really need any of this `ok2run` stuff here. If it can't import `matplotlib` then a useful enough exception will be raised anyway.